resources/lang/en/cachet.php

```diff
@@ -93,7 +93,7 @@
             'manage_subscription' => 'We\'ve sent you an email, please click the link to manage your subscription',
             'subscribe'           => 'Subscribe to email updates.',
             'subscribed'          => 'You\'ve been subscribed to email notifications, please check your email to confirm your subscription.',
-            'updated-subscribe'   => 'You\'ve succesfully updated your subscriptions.',
+            'updated-subscribe'   => 'You\'ve successfully updated your subscriptions.',
```

Comment:
I think this one needs to be reverted!

```suggestion
            'updated-subscribe'   => 'You\'ve succesfully updated your subscriptions.',
```